File: astorWork/Root-dev-Oct2018/astorwork-web-ui/src/app/material-track/classes/qc-defect.ts
import { Moment } from "moment";

export class QCDefect{
    id: number;
    remarks: string;
    isOpen: string;
    createdBy: string;
    createdDate:  Moment;
    updatedBy: string;
    updatedDate:  Moment;
    countPhotos: number;
    caseID: number;
}

File: astorWork/Root-dev-Oct2018/astorWorkMobile/astorWorkMobile/astorWorkMobile/MaterialTrack/Entities/QCDefect.cs
using System;

namespace astorWorkMobile.MaterialTrack.Entities
{
    public class QCDefect
    {
        public int ID { get; set; }
        public string Remarks { get; set; }
        public string CreatedBy { get; set; }
        public DateTimeOffset CreatedDate { get; set; }
        public string UpdatedBy { get; set; }
        public DateTimeOffset? UpdatedDate { get; set; }
        public bool IsOpen { get; set; }
        public bool IsClosed { get { return !IsOpen; } }
        public int CountPhotos { get; set; }
    }
}
